### PR TITLE
refactor(ui): remove useless code and dependencies

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
- Add some tooling to show bundle stats.
- Switch to lodash-es.
- Custom webpack to keep only fr and en locales in moment.
- Import minified jquery and semantic.
- Refact some existing views using data-table component.

1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-engine-cli-tools#9594.0 ✘  
* Build minimal
  * cdsctl linux ✔ 
  * check SQL Files number ✔ 
  * Engine Build linux amd64 ✔ 
  * tools/debpacker ✘  
  * tools/os-ansible-inventory ✔ 
  * Worker linux amd64 / 386 ✔

File: ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts [hunk 1]
```diff
@@ -1,36 +1,51 @@
...
-  constructor(private _modalService: SuiModalService) {
+  constructor(
+    private _modalService: SuiModalService,
+    private _theme: ThemeStore
+  ) {
     this.codeMirrorConfig = {
-      matchBrackets: true,
-      autoCloseBrackets: true,
-      mode: 'application/json',
+      tchBrackets: true,
```
Comment (bnjjj) [inline ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts]: are you drunk ?

File: ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts [hunk 2]
```diff
@@ -1,36 +1,51 @@
...
+      autoCloeBrackets: true,
```
Comment (bnjjj) [inline ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts]: are you drunk ?

File: ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts [hunk 3]
```diff
@@ -1,36 +1,51 @@
...
+      de: 'application/json',
```
Comment (bnjjj) [inline ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts]: same

File: ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts [hunk 4]
```diff
@@ -1,36 +1,51 @@
...
       lineWrapping: true,
       autoRefresh: true,
-      readOnly: true
+      adOnly: true
```
Comment (bnjjj) [inline ui/src/app/shared/workflow/node/hook/details/hook.details.component.ts]: same

**Pull request merged**